upapi/user/__init__.py

```diff
@@ -29,8 +30,18 @@ def __init__(self, *args, **kwargs):
     @property
     def friends(self):
         """
+        If not cached yet, cache the Friends object. Return the Friends object.
+
+        :return: a Friends object
+        """
+        if self._friends is None:
+            self._friends = self.get_friends()
+        return self._friends
```

Comment:
Ok, lastest commit has get_friends setting _friends as well, so the answer to your question is to just call get_friends()